### PR TITLE
fix: txn cache roundtrip integration test

### PR DESCRIPTION
**Pull request by anna-carroll**

### CODE REVIEW

**Comment by anna-carroll** [comment]
the integration tests don't actually run locally, i don't know if we're actually running them anywhere? cc maybe @rswanson ?

File: crates/builder/tests/tx_poller_test.rs [hunk 1]
```diff
@@ -12,24 +12,31 @@ mod tests {
...
+        let client = reqwest::Client::new();
+
         // create a new signed test transaction
         let wallet = PrivateKeySigner::random();
         let tx_envelope = new_test_tx(&wallet);
 
         // send that transaction to ensure there is at least one tx in pool to parse
         let _ = client
-            .post(config.tx_pool_url.to_string() + "/add")
+            .post(config.tx_pool_url.to_string() + "add")
```
[the remark this answers is not in the document]
Comment (anna-carroll) [reply crates/builder/tests/tx_poller_test.rs:35]: should i change the way im handling this to add the slash back in the code and remove it from the config?

File: crates/builder/tests/tx_poller_test.rs [hunk 2]
```diff
@@ -12,24 +12,31 @@ mod tests {
...
         // create a new test environment
-        let client = reqwest::Client::new();
         let (_, config) = setup_test_builder().await;
 
+        // post a tx to the cache
+        post_tx(config.clone()).await;
+
+        // assert that we parsed at least one transaction
+        let got = tx_poller::TxPoller::new(&config).check_tx_pool().await.unwrap();
+        assert!(!got.is_empty());
```
Comment (dylanlott) [inline crates/builder/tests/tx_poller_test.rs:22]: 🎀

File: crates/builder/tests/tx_poller_test.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (dylanlott) [reply crates/builder/tests/tx_poller_test.rs:35]: I would pull the trailing slash off the base URL and leave it on the `/get` and `/add` for example.

**Comment by anna-carroll** [comment]
@dylanlott okay, i updated the code here to keep the slash on and remove it from the config. we'll need to update the infra config too. i can do that now

**Review by dylanlott — APPROVED** [review]
LGTM 🎀

**Pull request merged**